superset/assets/src/dashboard/actions/dashboardState.js

```diff
@@ -258,6 +258,27 @@ export const TOGGLE_BUILDER_PANE = 'TOGGLE_BUILDER_PANE';
 export function toggleBuilderPane() {
   return { type: TOGGLE_BUILDER_PANE };
 }
+export const MODAL_CHART_UPDATE_SUCCEEDED = 'MODAL_CHART_UPDATE_SUCCEEDED';
+export function updateModalChartId(id) {
```

Comment:
rename this function as per action type or change action type.

Reply:
renamed it to UPDATE_MODAL_CHART_ID